utilities/tools/ConvertTo-ARMTemplate.ps1

```diff
@@ -66,13 +66,15 @@ if ($ConvertChildren) {
 #region Remove existing deploy.json files
 Write-Verbose 'Remove existing deploy.json files'
 
-$JsonFilesToRemove = Get-ChildItem -Path $armFolderPath -Filter 'deploy.json' -Recurse -Force -File
-Write-Verbose "Remove existing deploy.json files - Remove [$($JsonFilesToRemove.count)] file(s)"
-if ($PSCmdlet.ShouldProcess("[$($JsonFilesToRemove.count)] deploy.json files(s) in path [$armFolderPath]", 'Remove-Item')) {
-    $JsonFilesToRemove | Remove-Item -Force
+if (Test-Path -Path (Join-Path -Path $armFolderPath -ChildPath 'deploy.bicep')) {
+    $JsonFilesToRemove = Get-ChildItem -Path $armFolderPath -Filter 'deploy.json' -Recurse -Force -File
+    Write-Verbose "Remove existing deploy.json files - Remove [$($JsonFilesToRemove.count)] file(s)"
+    if ($PSCmdlet.ShouldProcess("[$($JsonFilesToRemove.count)] deploy.json files(s) in path [$armFolderPath]", 'Remove-Item')) {
+        $JsonFilesToRemove | Remove-Item -Force
+    }
+    Write-Verbose 'Remove existing deploy.json files - Done'
```

Comment:
Added Test-Path to only delete json files if the bicep file exist, ensuring that the conversion runs clean. Also allowing customers to bring in new modules, running the script again and only having it convert the new module without 'destroying' the entire repo (removing all deploy.json files)